### PR TITLE
bump ignition from v0.7.0 to v0.13.0

### PR DESCRIPTION
**Pull request by pbx0**
This was part of a PR i'm abandoning for more then one reason. I no longer need this PR but since the work is done I figured i'd see if anyone else has interest in this bump.

### CODE REVIEW

File: platform/conf/conf.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ type Conf struct {
 func New(userdata string) (*Conf, error) {
 	c := &Conf{}
 
-	ignc, err := v2.Parse([]byte(userdata))
+	ignc, _, err := v2.Parse([]byte(userdata))
```
Comment (euank) [inline platform/conf/conf.go]: I think we'd want to log the report if there's an err for the convenient of the test author.

Comment (pbx0) [inline platform/conf/conf.go]: Should I loop through each entry and convert it to a string and then log it if there is an error? It wasn't clear to me what a report tells me. https://github.com/coreos/ignition/blob/master/config/validate/report/report.go

**Comment by bgilbert** [comment]
@pbx0 This would be useful for some work I'm doing.  Do you have time to finish it, or would you like me to take a look?

**Comment by pbx0** [comment]
@bgilbert Feel free to take it over, I'm not sure I will get back around to this in time. Looks like ignition has one newer release now as well.

**Comment by bgilbert** [comment]
@pbx0 Sounds good.  Closing in favor of #576.

**Pull request closed without merging**